--- CMake/Packages/FindOGRE.cmake.orig	2011-05-08 20:28:04.000000000 +0200
+++ CMake/Packages/FindOGRE.cmake	2011-11-26 13:14:53.065829030 +0100
@@ -144,7 +144,7 @@
 # try to locate Ogre via pkg-config
 use_pkgconfig(OGRE_PKGC "OGRE${OGRE_LIB_SUFFIX}")
 
-if(NOT OGRE_BUILD_PLATFORM_IPHONE)
+if(APPLE AND NOT OGRE_BUILD_PLATFORM_IPHONE)
   # try to find framework on OSX
   findpkg_framework(OGRE)
 else()
